gapic-generator/templates/default/helpers/presenters/gem_presenter.rb

```diff
@@ -118,4 +120,10 @@ def gem_config key
 
     @api.configuration[:gem][key]
   end
+
+  def files_blacklist
+    gem_config(:blacklist) ||
```

Comment:
I am not 100% happy with this configuration name, but I couldn't think of anything better.

Right now this only contains proto files that should be skipped, but we could also put other values in here. For example if we wanted to skip Longrunning messages other than Operation, we could list them in the config. I don't know if we need separate lists for files and messages, so I left the name generic.

Reply:
Right now it's for proto files so I'd go with something more explicit like `proto_file_blacklist` or something similar.

Reply:
YAML can be nested, so I would move this to `gem/blacklist/protos`, and we can add something like `gem/blacklist/messages` if we want to later.